### DISCUSSION TITLE
Is there a way to make ClickHouse include Peak Memory Usage in the query's `x-clickhouse-summary` response?

### DISCUSSION THREAD

**Discussion by Tseyang** [ClickHouse/ClickHouse#56820] [Q&A]
Hi, I'm wondering if its possible to configure ClickHouse to include peak memory usage (the info you can see when you run a query with `set send_logs_level = 'debug'`) in the response header for `x-clickhouse-summary` like it does for `result_rows`, `read_bytes`, etc.?

**Comment by tavplubix** [comment]
The problem is that it will not work without `wait_end_of_query=1` ([link](https://clickhouse.com/docs/en/integrations/python#settings-argument))
However, `x-clickhouse-summary` doesn't work without it too

**Comment by den-crane** [comment]
it's not possible to configure. 
This feature was removed because of the bug https://github.com/ClickHouse/ClickHouse/pull/55336
